src/apis/AbstractApi.ts

```diff
@@ -6,4 +6,10 @@ export abstract class AbstractApi {
   constructor(restClient: RestClient) {
     this._restClient = restClient;
   }
+
+  protected validateId(id: string) {
```

Comment:
🙃 
How about a more generic "isRequired" ?

Reply:
@igneel64 Are you thinking of a more generic validation? There's a lot of different ways we can do this, and I wasn't sure where to draw the line.

The `isRequired` name implies a boolean outcome (true or false). It could become generic enough to check for any string value presence `function isRequired(value: string): boolean`. Although then a more appropriate name would be `isNotBlank`.

I wanted to throw the error as part of the check, so all the functionality is in the abstract class and descendants can just call this method.

Do you have any idea how we could improve this if you agree with the points above?

Reply:
@gkats 
Yes actually I was thinking of a more validation like thing, but I also don't want to make this a 'validation PR' which is not the scope.
IMO let's leave it as is for now since it is a production bug. WDYT ?

Reply:
Agreed. I could change the name of the function to `requireId` instead of `validateId` since it's probably more descriptive.